pyanaconda/ui/gui/spokes/installation_source.py

```diff
@@ -1081,6 +1107,14 @@ def refresh(self):
         # Update the URL entry validation now that we're done messing with sensitivites
         self._update_url_entry_check()
 
+        # if subscription module is available we might need to refresh the label
+        # of the CDN/Satellite radio button
+        if self._subscription_module:
+            if self.registered_to_satellite:
+                self._cdn_button.set_label(C_("GUI|Software Source", "_Satellite"))
+            else:
+                self._cdn_button.set_label(C_("GUI|Software Source", "Red Hat _CDN"))
```

Comment:
I'll expand the docstring - basically its about the important disctinction between where the repositories are hosted for "normal" vs "Satellite" registration. 

If subscription module is available we might need to refresh the label
of the CDN/Satellite radio button, so that it properly describes what is providing the repositories available after registration.

For registration to Red Hat hosted infrastructure (also called Hosted Candlepin) the global Red Hat CDN efficiently provides quick access to the repositories to customers across the world over the public Internet.

If registered to a customer Satellite instance, it is the Satellite instance itself that provides the software repositories.

This is an important distinction as Satellite instances are often used in environments not connected to the public Internet, so seeing the installation source being provided by Red Hat CDN which the machine might not be able to reach could be very confusing.